frontend/src/lib/forms/helpers.ts

```diff
@@ -2,7 +2,7 @@ import { type SuperValidated } from 'sveltekit-superforms';
 import { z } from 'zod';
 
 /**
- * Sets the settings in memory in the backend.
+ * Sets the settings in memory in the 

```

Comment:
Whats happened here?

Reply:
VS Code Global search and replace 😂. I'll fix lol

Everytime I was coding, copilot tried to import things with `backend.` in it instead of `program.`